library/src/scala/tasty/reflect/SourceCodePrinter.scala

```diff
@@ -1157,12 +1162,14 @@ class SourceCodePrinter[R <: Reflection & Singleton](val reflect: R)(syntaxHighl
             printFullClassName(tp)
```

Comment:
```suggestion
            if printTypePrefix then
              printFullClassName(tp)
            else ??? // print class name 
```

Reply:
Or maybe change the logic of `printFullClassName`.

Reply:
I added a guard inside `printFullClassName`